README.md

```diff
@@ -22,10 +22,11 @@
 
 ## Links
 
-- **[Documentation](https://mochajs.org/)**
+- **[Documentation](https://mochajs.org)**
```

Comment:
A nitpick on my end: unnecessary trailing `/` slashes at the end of URLs irritate me. 😄

Reply:
Web server always assumes it's serving a file, unless the URL ends in a directory separator.

If the URL is to a directory, the server will append the directory index file (typically 'index.html') to the URL before it tries. If the server fails to return a file, it appends the directory separator to the path and retries as a directory.

Thus doing it this way causes *twice* the number of network calls needed for every directory URL without a trailing directory separator.

[Apache: Mapping URLs to Filesystem Locations #DocumentRoot](https://httpd.apache.org/docs/2.4/urlmapping.html#documentroot)

Reply:
@plroebuck Thats something that eg Apache does internally – it’s not something done through multiple requests – and eg plain Node.js doesn’t do anything like that at all

@JoshuaKGoldberg Not sure I like this though, the one with the slash in the end is the more “correct” one 😜

Reply:
Haha how do we define "correct" here... per _some_ specs the trailing slash is more correct. But end-users are generally told to visit URLs without saying the slash.